docs/src/utils/arch.md

```diff
@@ -0,0 +1,3 @@
+# arch
```

Comment:
why these files have to be in-tree ?
seems that they can be generated

Reply:
They can be (and are) generated indeed. The idea was that we might want to include more information here than in the help. Much like the difference between the GNU manual and `--help` on the utils. So you could write:

```markdown
# arch

Some long explanation about the usage of arch.

{{#include ../../_generated/arch-help.md }}

Some more explanation.
```

In the future this could become more fine-grained, so you can include the usage, options, etc. separately.

However, we could simplify it for now and focus on generation first, before adding the option to customize it more.

Reply:
maybe the long explanation could be added in the clap def
Otherwise, it might go out of sync quickly

Reply:
Alright, I'll simplify it to a single generated file per util!